extensions/amp-analytics/0.1/events.js

```diff
@@ -1079,7 +1080,7 @@ export class VideoEventTracker extends EventTracker {
       const isVisibleType = (type === VideoAnalyticsEvents.SESSION_VISIBLE);
       const normalizedType =
           isVisibleType ? VideoAnalyticsEvents.SESSION : type;
-      const details = /** @type {!VideoAnalyticsDetailsDef} */ (getData(event));
+      const details = /** @type {?JsonObject|undefined} */ (getData(event));
```

Comment:
@aghassemi i assume `details` cannot be a `string` type here

Reply:
correct. there are controlled custom events and `details` is generated by us